server/src/session.ts

```diff
@@ -876,33 +861,50 @@ export class Session {
    * Disable the language service if the specified `project` is not Angular or
    * Ivy mode is enabled.
    */
-  private checkProject(project: ts.server.Project) {
+  private disableLanguageServiceIfNecessary(project: ts.server.Project): void {
```

Comment:
hmm, I'm not sure I like the new name for this. The function does quite a bit more than disabling if necessary (i.e. running `ngcc`). Though I do agree that `checkProject` isn't the best either. At the very least, I think this could use a javadoc update at the top.

Reply:
@atscott ok, I figured it's better to just refactor the code to remove this function entirely. PTAL.